apps/accounts/models.py

```diff
@@ -10,7 +10,6 @@
 from django.db import models
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
-import boto3
```

Comment:
This was the last direct call to boto, so we were able to remove it from the requirements.in list